spec/models/info_request_batch_spec.rb

```diff
@@ -101,23 +101,21 @@
       end
     end
 
-    it 'should send requests to requestable public bodies, and return a list of unrequestable ones' do
-      allow(first_public_body).to receive(:is_requestable?).and_return(false)
-      unrequestable = info_request_batch.create_batch!
-      expect(unrequestable).to eq([first_public_body])
+    it 'should send requests to requestable public bodies' do
+      expect(info_request_batch).to receive(:requestable_public_bodies).
+        and_return([first_public_body])
+      info_request_batch.create_batch!
       expect(info_request_batch.info_requests.size).to eq(1)
       request = info_request_batch.info_requests.first
       expect(request.outgoing_messages.first.status).to eq('sent')
     end
 
     it 'should not only send requests to public bodies if already sent' do
```

Comment:
This spec description is hard to read (added in https://github.com/mysociety/alaveteli/pull/4666/commits/0ef4cfc8b4bc4f24b2d05ee739ac9a7ee5cf0834).

I _think_ this should read:

```diff
- it 'should not only send requests to public bodies if already sent' do
+ it 'does not resend requests to public bodies that have already received the request' do
```

Reply:
Changed in 590e1e9826eb6c365903ef6f1c3fa91f9809f444